modules/oe_content_event/modules/oe_content_event_person_reference/config/install/field.storage.node.oe_event_speakers.yml

```diff
@@ -14,7 +14,7 @@ settings:
 module: entity_reference_revisions
 locked: false
 cardinality: -1
-translatable: false
+translatable: true
```

Comment:
Why? We don't translate entity reference revision fields usually.

Reply:
Right.